modules/service/src/main/scala/lucuma/itc/legacy/codecs.scala

```diff
@@ -391,10 +392,10 @@ private given Decoder[SeriesDataType] = (c: HCursor) =>
     }
   }
 
-private given Decoder[ChartType] = (c: HCursor) =>
+private given Decoder[GraphType] = (c: HCursor) =>
   Decoder.decodeJsonObject(c).flatMap { str =>
-    val key = str.keys.headOption.orEmpty
-    Try(ChartType.valueOf(key)).toEither.leftMap { _ =>
+    val key: String = str.keys.headOption.orEmpty
+    Try(GraphType.valueOf(key.replace("Chart", "Graph"))).toEither.leftMap { _ =>
```

Comment:
Why the `replace`?

Reply:
We used `Graph` and `Chart` interchangeably in different places. I unified everything to `Graph`, which seems more accurate for this case.

However, the legacy API uses `Chart`. I'm planning to switch this renaming mechanism to something more solid though, with both legacy and new names defined for each graph type.